.tool-versions

```diff
@@ -1,2 +1,2 @@
-nodejs 14.19.3
-yarn 1.22.19
+nodejs 18.17.1
+pnpm 8.8.0
```

Comment:
Does it make sense to use latest pnpm version (` 8.14.3`)?

Reply:
Probably, I'll make sure it works OK.